source/Halibut/Diagnostics/EventType.cs

```diff
@@ -5,16 +5,39 @@ namespace Halibut.Diagnostics
     public enum EventType
     {
         OpeningNewConnection,
+        [Obsolete]
```

Comment:
I flagged these as obsolete because we don't actually use them. We can clean them up when we remove all the other obsolete flags